guides/source/active_model_basics.md

```diff
@@ -506,35 +1262,45 @@ Finished in 0.024899s, 240.9735 runs/s, 1204.8677 assertions/s.
 6 runs, 30 assertions, 0 failures, 0 errors, 0 skips
 ```
 
-An object is not required to implement all APIs in order to work with
-Action Pack. This module only intends to guide in case you want all
-features out of the box.
-
 ### SecurePassword
 
-`ActiveModel::SecurePassword` provides a way to securely store any
-password in an encrypted form. When you include this module, a
-`has_secure_password` class method is provided which defines
-a `password` accessor with certain validations on it by default.
+`ActiveModel::SecurePassword` provides a way to securely store any password in
+an encrypted form. When you include this module, a `has_secure_password` class
+method is provided which defines a `password` accessor with certain validations
+on it by default.
 
 #### Requirements
 
-`ActiveModel::SecurePassword` depends on [`bcrypt`](https://github.com/codahale/bcrypt-ruby 'BCrypt'),
-so include this gem in your `Gemfile` to use `ActiveModel::SecurePassword` correctly.
-In order to make this work, the model must have an accessor named `XXX_digest`.
-Where `XXX` is the attribute name of your desired password.
-The following validations are added automatically:
+`ActiveModel::SecurePassword` depends on
+[`bcrypt`](https://github.com/codahale/bcrypt-ruby 'BCrypt'), so include this
+gem in your `Gemfile` to use `ActiveModel::SecurePassword`.
+
+```ruby
+gem "bcrypt"
+```
 
-1. Password should be present.
-2. Password should be equal to its confirmation (provided `XXX_confirmation` is passed along).
+`ActiveModel::SecurePassword` requires you to have a `password_digest`
+attribute.
+
+The following validations are added automatically:
+1. Password must be present on creation.
+2. Confirmation of password (using a `password_confirmation` attribute)
 3. The maximum length of a password is 72 bytes (required as `bcrypt`, on which
-   ActiveModel::SecurePassword depends, truncates the string to this size before encrypting it).
+   ActiveModel::SecurePassword depends, truncates the string to this size before
+   encrypting it).
```

Comment:
We've just mentioned above that bcrypt is a requirement of SecurePassword, maybe we can simplify this a bit here by removing that mention?

```suggestion
3. The maximum length of a password is 72 bytes (required as `bcrypt` truncates
   the string to this size before encrypting it).
```